apps/backend/src/datasources/postgres/TemplateDataSource.ts

```diff
@@ -720,6 +720,15 @@ export default class PostgresTemplateDataSource implements TemplateDataSource {
     question: string,
     default_config: string
   ): Promise<Question> {
+    const naturalKeyExistAlready = await database
+      .select('natural_key')
+      .from('questions')
+      .where('natural_key', natural_key);
+
+    if (naturalKeyExistAlready.length != 0) {
+      natural_key = natural_key.concat('_copy');
+    }

```

Comment:
All looking good just, but is there a chance that a question can belong to two questionnaires and by some chance we try to import them one after another. This means appending _copy alone might not work but maybe add an additional random generated word or number for example a timestamp.